libcxx/test/std/utilities/format/format.functions/bug_81590.compile.pass.cpp

```diff
@@ -0,0 +1,37 @@
+//===----------------------------------------------------------------------===//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: GCC-ALWAYS_INLINE-FIXME
```

Comment:
I really doubt this is required by this test. Is it?

Reply:
This is indeed not needed. The FreeBSD CI failure is unrelated. Thanks for the review!